src/server/models/url.ts

```diff
@@ -182,6 +178,15 @@ export const Url = <UrlTypeStatic>sequelize.define(
             transaction: Sequelize.Transaction
           },
         )
+
+        await UrlClicks.upsert(
+          {
+            shortUrl: url.shortUrl,
+          },
+          options as Sequelize.CreateOptions & {
+            transaction: Sequelize.Transaction
+          },
+        )
```

Comment:
using `upsert` is necessary during this transition phase because the DB triggers are still in place, and this step clashes with what the trigger is trying to do. Once the trigger is removed, we can change `upsert` to `create` if we want to

Reply:
file an issue and add a TODO?

Reply:
on another note, should the typecasting of `options as Sequelize.CreateOptions` be revisited?

Reply:
agreed, it will still pass typescript checks even without typecasting, and I added this to be consistent with `writeToUrlHistory`. This can be changed together in a subsequent PR when we change `upsert` to `create`.

Reply:
ok, please include in issue